src/world_interface/noop_world.cpp

```diff
@@ -8,7 +8,16 @@ using namespace robot::types;
 
 namespace robot {
 
-const WorldInterface WORLD_INTERFACE = WorldInterface::noop;
+extern const WorldInterface WORLD_INTERFACE = WorldInterface::noop;
+
+const DiffDriveKinematics driveKinematics() {
+	//doesn't really matter what we set this to
+	return DiffDriveKinematics(1); 
+}
+
+const DiffWristKinematics wristKinematics() {
+	return DiffWristKinematics();
+}
```

Comment:
Maybe these should return references to a statically allocated kinematics object? That would reduce a lot of unnecessary copying.